doc/docs/api/description.md

```diff
@@ -41,6 +41,10 @@ Note that all properties also come with change events, so you can easily track w
 
     Disables the collection of metrics on all agents.
 
+  * `SetLogLevel(in s log_level)`
+
+    Set new log level for hirte manager. This change is persistent as long as hirte not restarted.
```

Comment:
I'd change to `Set the new log level ...`.
And also shouldn't we use the word *controller* instead of *manager*?